bika/lims/utils/__init__.py

```diff
@@ -761,4 +760,17 @@ def check_permission(permission, obj):
     """
     mtool = api.get_tool('portal_membership')
     object = api.get_object(obj)
-    return mtool.checkPermission(permission, object)
\ No newline at end of file
+    return mtool.checkPermission(permission, object)
+
+def to_int(value, default=0):
+    """
+    Tries to convert the value passed in as an int. If no success, returns the
+    default value passed in
+    :param value: the string to convert to integer
+    :param default: the default fallback
+    :return: int representation of the value passed in
+    """
+    try:
+        return int(value)
+    except (TypeError, ValueError):
+        return to_int(default, default=0)
```

Comment:
Please keep in mind that this always return 0 if inserted something like `to_int("foo", default="bar")`

Reply:
Yes, its fine